tests/Browser/ManageAccountFilterTest.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace Tests\Browser;
+
+use App\Enum\UserRoleEnum;
+use App\Models\User;
+use Facebook\WebDriver\WebDriverKeys;
+use Laravel\Dusk\Browser;
+use Tests\DuskTestCase;
+
+class ManageAccountFilterTest extends DuskTestCase
+{
+    private $admin;
+
+    private function createUsers()
+    {
+        $this->admin = User::factory()->create([
+            'email' => 'admin',
+            'password' => 'password',
+        ])->assignRole('admin');
+
+        User::factory()->create([
+            'email' => 'aaa',
+            'password' => 'password',
+        ])->assignRole('user');
+
+        User::factory()->create([
+            'email' => 'zzz',
+            'password' => 'password',
+        ])->assignRole('user');
+
+        User::factory()->create([
+            'email' => 'aab',
+            'password' => 'password',
+        ])->assignRole('admin');
+
+        User::factory()->create([
+            'email' => 'abzzza',
+            'password' => 'password',
+        ]);
+    }
+
+    /**
+     * @group manage
+     * @return void
+     */
+    public function test_sorting_by_email()
```

Comment:
Naar mijn idee slaagt deze test altijd, omdat 'aaa' en 'zzz' altijd op de pagina bevinden (er worden 5 users gemaakt en er staan max 10 users per pagina). Ik had zelf de clicklinks weggehaald en toen slaagde de test ook.

Reply:
Alle seeders worden ook altijd eerst gedraaid bij het testen, daar zitten veel users in en ik heb met screenshots net getest en daar is dit te zien voor het filteren:
![image](https://github.com/Avans-290IN4SOj-Scouting/Scouting/assets/130562052/d40cf5b1-16a9-46cc-9463-ad2358c376ce)
En dit er na:
![image](https://github.com/Avans-290IN4SOj-Scouting/Scouting/assets/130562052/f9972f5d-0a96-4b63-9780-3892f668af40)

Reply:
hmmm, ik zal hier nog eens naar kijken